Automattic/wp-calypso
# Stepper: Added flow logic for Woo transfer/plugin install

#### Changes proposed in this Pull Request

* Added flow logic for Woo transfer/plugin install
* Added placeholder steps

#### Testing instructions


1. Transfer
* Using a **non-atomic** site on a paid plan, enable the `stepper-woocommerce-poc` flag and go to the Woo flow starting on `/setup/intent?siteSlug=<slug>&flags=stepper-woocommerce-poc`
* You should see the placeholder for the site Transfer
2. Plugin install
* Using an **atomic** site on a paid plan, enable the `sepper-woocommerce-poc` flag and go to the Woo flow starting on `/setup/intent?siteSlug=<slug>&flags=stepper-woocommerce-poc`
* You should see the placeholder for the plugin install



Related to #62718
